desc/integrals/bounce_integral.py

```diff
@@ -145,7 +145,7 @@ class Bounce2D(Bounce):
     Notes
     -----
     Magnetic field line with label α, defined by B = ∇ψ × ∇α, is determined from
-      α : ρ, θ, ζ ↦ θ + λ(ρ,θ,ζ) − ι(ρ) [ζ + ω(ρ,θ,ζ)]
+      α : ρ, θ, ζ ↦ θ + Λ(ρ,θ,ζ) − ι(ρ) [ζ + ω(ρ,θ,ζ)]
```

Comment:
I am confused again by this equation. Specifically the + sign before omega.
Is the convention zeta = phi + omega or zeta = phi - omega, where phi is the cylindrical angle and zeta is the generalized angle.

Reply:
The latter. One can control +f search desc/compute/_core.py for `name="phi"` and `name="theta_PEST"` to see the definitions.